src/CarbonAware.WebApi/src/Controllers/CarbonAwareController.cs

```diff
@@ -146,6 +146,42 @@ public async Task<IActionResult> GetCurrentForecastData([FromQuery(Name = "locat
         }
     }
 
+    /// <summary>
+    /// Given an array of requested historical forecasts, retrieve the forecasted data and calculate the optimal
```

Comment:
I'm still confused by this api, given the fact that now the consumer has to provide an array of multiple combinations (locations + date times) which makes is difficult to understand. Perhaps starting small (one location on a specific date range), and walkthrough the use case would help to know the intent of this.

Reply:
Hi @juzuluag, I had a chat with @esgraham , and while we agree your question is valid, the proposed spec change that allows multiple locations to be passed in as a Batch request, we feel does match the 'current approach' of the other APIs in the Controller, which support requesting multiple locations, and so think we should accept this proposal -- it can always be (re)debated + changed in a future sprint